samples/resources/README.md

```diff
@@ -0,0 +1,87 @@
+---
+page_type: sample
+languages:
+- python
+products:
+- azure
+description: "These samples will show you how to manage reosurce in resource sdk."
```

Comment:
These code samples will show you how to manage Resources using Azure SDK for Python